migrator/migrations/m_168_to_m_169_postgres_remove_clustercve_permission/migration.go

```diff
@@ -52,7 +53,7 @@ func propagateAccessForPermission(permission string, accessLevel storage.Access,
 }
 
 func cleanupPermissionSets(db postgres.DB) error {
-	ctx := context.Background()
+	ctx := sac.WithAllAccess(context.Background())
```

Comment:
✅ 

We should make this a product-wide recommendation for all internally used contexts.